Nos/Views/SettingsView.swift

```diff
@@ -51,27 +44,13 @@ struct SettingsView: View {
         Form {
             Section {
                 HStack {
-                    SecureField(String(localized: .localizable.privateKeyPlaceholder), text: $privateKeyString)
+                    Text("•••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••")
```

Comment:
63 dots since that's what the old one showed. 🤷

Reply:
I see what you are doing. It makes no sense to use a secure field if we always show the same thing.
However, you can use `String(repeating: "•", count: 63)` instead.